src/main/java/core/basesyntax/write/WriteFile.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.write;
+
+import java.io.IOException;
+
+public class WriteFile implements FileWriter {
+    private static final String HEAD_OF_FILE = "fruit, quantity\n";
+
+    public void writingToFile(String path, String totalFruit) {
+        try (java.io.FileWriter writer = new java.io.FileWriter(path, false)) {
```

Comment:
```suggestion
        try (FileWriter writer = new FileWriter(path, false)) {
```
always import all necessary classes